drivers/bus/acpi/cmbatt/cmbatt.h

```diff
@@ -67,6 +67,8 @@ typedef enum _CMBATT_EXTENSION_TYPE
 #define CM_MAX_VALUE                0x7FFFFFFF
 #define CM_UNKNOWN_VALUE            0xFFFFFFFF
 
+#define CONVERT_MAH_TO_MWH(mA, Volts) ((mA * Volts + 500) / 1000)
```

Comment:
```suggestion
#define CONVERT_MAH_TO_MWH(mA, Volts) (((mA) * (Volts) + 500) / 1000)
```

Reply:
The suggestion still remains to be applied, otherwise using the macro may be unsafe if it gets used like this: `CONVERT_MAH_TO_MWH(something, voltage + delta)`.

Reply:
It already is?
![Capture](https://github.com/user-attachments/assets/541a8cf5-3ece-47ad-9827-5a9562a35ac6)